src/pages/DetailPage.js

```diff
@@ -0,0 +1,84 @@
+import PropTypes from "prop-types";
+import styled from "styled-components/macro";
+import Heading from "../components/Header";
+import Button from "../components/Button";
+import DividingLine from "../components/DividingLine";
+
+DetailPage.propTypes = {
+  concert: PropTypes.object,
+  onNavigate: PropTypes.func,
+  pageName: PropTypes.string,
+};
+
+export default function DetailPage({ concert, onNavigate, pageName }) {
+  return (
+    <div>
+      <Heading pageName={pageName} /> <DividingLine />
```

Comment:
Do you need two `DividingLine`s? You already have one in the `Heading` component anyways.
```suggestion
      <Heading pageName={pageName} /> 
      <DividingLine />
```

Reply:
Oh, thats right! I only need one here. At first I only created the DivingLine component without putting it into the Heading. So at this point I forgot to delete it.